### ISSUE TITLE
Translation problems for BR

### ISSUE THREAD

**Issue by Myiamoto86** [Stephan-S/FS22_AutoDrive#603]
Guys, good afternoon.

I hereby inform you that the BR translation is having problems... I had to get the translation file from the previous version and adapt it to the current version to be able to use it in the BR language.

**Comment by Iwan1803** [comment]
There was Pull request was for that language fill which will be integrated in the next version.

**Comment by Myiamoto86** [comment]
![image](https://user-images.githubusercontent.com/108041428/180674370-04fb0518-9dee-40d9-aa3e-8029161b89d7.png)
Dear Sirs,
![Farming Simulator 22 Screenshot 2022 07 24 - 22 11 55 89](https://user-images.githubusercontent.com/108041428/180675082-8925a0db-09f3-471e-88eb-901edfc64a58.png)

Good evening.

It took a little work, but I managed to refine translation_br.xml to make it more understandable to Brazilian players. 
In the HUD, I needed to make changes and not translation, because, due difficult understanding during translation of the HUD help "?", according to attached image.
I hope you consider, insert this translation for a next update.
Regards.
